src/plugins/reporter-bugzilla.c

```diff
@@ -313,6 +331,19 @@ int main(int argc, char **argv)
     export_abrt_envvars(0);
 
     map_string_t *settings = new_map_string();
+    problem_data_t *problem_data;
+
+    /* pull in some defaults from os-release */
+    problem_data = create_problem_data_for_reporting(dump_dir_name);
+    if (!problem_data)
+        xfunc_die(); /* create_problem_data_for_reporting already emitted error msg */
+    else
+    {
+        map_string_t *osinfo = new_map_string();
+        problem_data_get_osinfo(problem_data, osinfo);
+        set_default_settings(osinfo, &settings);
```

Comment:
oh dear, I'll get a fix pushed up ASAP.  I'll hold off squashing for now due to conversation on `parse_release.c`